static/game/crosser.js

```diff
@@ -0,0 +1,65 @@
+import { Enemy } from "./enemy.js";
+import { Vector } from "../modules/vector.js";
+import { polygon } from "./draw.js";
+import { getTotalTime } from "../modules/gamemanager.js";
+
+export class Crosser extends Enemy {
```

Comment:
Add a comment explaining what a crosser is/does, since the name isn't very descriptive